### PR TITLE
[Service worker] Add MIME check for importScript 

### PR DESCRIPTION
**Pull request by sherryzy**

### CODE REVIEW

**Comment by kaidokert** [comment]
Is this a cherry-pick of a previous CL ? Commit history looks a bit strange, could you make sure you update to the tip of `origin/23.lts.1+` first before applying the cherry-pick ?

**Comment by sherryzy** [comment]
> Is this a cherry-pick of a previous CL ? Commit history looks a bit strange, could you make sure you update to the tip of `origin/23.lts.1+` first before applying the cherry-pick ?

It is a cherry-pick of a few previous Gerrit CL. I add all the three CLs in one PR. Not sure if it's the ideal way. I sync the fork branch in Github UI. Not sure why the old commits (the font one) still show up here. Let me try again.

**Comment by sherryzy** [comment]
Clean the Commit history.

**Review by jellefoks — APPROVED** [review]
